templating/twig_extension.rst

```diff
@@ -15,8 +15,12 @@ your code faster.
 
 .. tip::
 
-    Before writing your own extensions, have a look at the
-    `Twig official extension repository`_.
+    When writing your own extensions, you might want to learn
+    from having a look at the `Twig Bridge`_  which contains most of
+    the extensions provided by the Symfony Framework.
+
+    We also have :doc:`a short article </reference/using_twig_extension_repository>`
```

Comment:
maybe we should also rename the new file to something shorter like `/templating/twig_extension_repository`